scripts/deduplicate_taxon.py

```diff
@@ -0,0 +1,64 @@
+# This script deduplicate existing entries in the existing taxon table by:
+#
+# - Load its configuration ("row X should be replaced by row Y") from an external file
+# - Based on that configuration, updates the FK that point to those records in multiple tables => each relationship to X is replaced by a FK to Y
+# - Finally, deletes the X entries in taxon
+
+# This gives us cleaner data to import from, and avoid errors later down the road (since our new "scientificname"
+# table rejects duplicate, thus breaking the script)
+#
+# This script updates the existing data, so while it can be run multiple times, it will only have effects the first time.
+#
+# After succesful execution, the following query should return no results (no more duplicates on scientificanme + authorship in the taxon table):
+# WITH cte_duplicates AS (
+#     SELECT acceptedname, scientificnameauthorship, COUNT(*) FROM biodiv.taxon
+#     GROUP BY acceptedname, scientificnameauthorship
+#     HAVING COUNT(*) > 1)
+#
+# SELECT t.* FROM
+# biodiv.taxon t, cte_duplicates
+# WHERE t.acceptedname = cte_duplicates.acceptedname AND
+#       t.scientificnameauthorship = cte_duplicates.scientificnameauthorship ORDER by t.acceptedname
+#
+# key: old taxon_id (to be deleted)
+# value: new taxon_id (to replace the other one)
+import os
+
+from helpers import get_database_connection, execute_sql_from_jinja_string, get_config
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def deduplicate_taxon(conn, config_parser):
+    import json
```

Comment:
Shouldn't this be moved out of the function and be put together with `import os`?

Reply:
Absolutely, it stayed there after a quick and dirty experiment, I'll move it!